dashboard/dashboard/views.py

```diff
@@ -125,6 +126,15 @@ def view_administration(request):
         context['plugins'] = plugin_client.get_plugins_info()
         context['job_type_versions'] = plugin_client.get_job_type_versions()
         context['infrastructure_targets'] = plugin_client.get_infrastructure_targets()
+
+        #Collect instances running on the same infrastructure
+        _infrastructure_instances: dict = collections.defaultdict(list)
+        for instance, infrastructure in context['infrastructure_targets'].items():
+            _versioned_name: str = infrastructure['name'] + ' (' + infrastructure['version'] + ')'
+            _infrastructure_instances[_versioned_name].append(instance)
+        infrastructure_instances: dict[str, List[str]] = sorted(_infrastructure_instances.items())
```

Comment:
Since you've sorted it, the type is different:
```suggestion
        infrastructure_instances: list[tuple[str, List[str]]] = sorted(_infrastructure_instances.items())
```